test/tests/jobs/CreateJobsTest.cpp

```diff
@@ -34,6 +35,17 @@ struct CreateJobsTest : tpunit::TestFixture {
         ASSERT_NOT_EQUAL(jobID1, jobID2);
     }
 
+    void createWithHttp() {
+        SData command("CreateJobs / HTTP/1.1");
+        command["jobs"] = "[{\"name\":\"testCreate\", \"data\":{\"blabla\":\"blabla\"}, \"repeat\": \"SCHEDULED, +1 HOUR\"}, {\"name\":\"testCreate2\", \"data\":{\"nope\":\"nope\"}}]";
```

Comment:
Instead of constructing the json by hand, can you use `STable` construct the json and then `SComposeJSONObject` to create json string

Reply:
This also makes it easier to query the db for the jobs and test the response